corehq/apps/reports/graph_models.py

```diff
@@ -12,6 +12,11 @@ def __init__(self, label=None, format=None, dateFormat=None):
         self.format = format
         self.dateFormat = dateFormat
 
+    def to_json(self):
+        return {'label': self.label,
+                'format': self.format,
+                }
```

Comment:
is dateFormat not necessary to pass down to the JS?

Reply:
True, it wasn't necessary to fix this, but I added it here:
https://github.com/dimagi/commcare-hq/pull/20884